test/ao/sparsity/test_pruner.py

```diff
@@ -238,7 +238,7 @@ def _check_pruner_valid_before_step(self, model, pruner, device):
                 assert module.parametrizations.weight[0].pruned_outputs == set()
 
     def _check_pruner_valid_after_step(self, model, pruner, pruned_set, device):
-        for config in pruner.module_groups:
+        for config in pruner.tensor_groups:
```

Comment:
wdyt, if we just use "groups"? Or is it too confusing?

Reply:
sure, I can do that.